domain/user.go

```diff
@@ -7,19 +7,27 @@ import (
 type User struct {
 	ID       uuid.UUID
 	Name     string
-	RealName string // TODO: 後で消す
+	realName string // private field
```

Comment:
このコメントは見れば分かるのでなくてもよさそう
```suggestion
	realName string
```